test/instrumentation/modules/@aws-sdk/fixtures/use-client-s3.js

```diff
@@ -228,7 +267,8 @@ function main () {
       s3Client.destroy()
       process.exitCode = 0
     },
-    function () {
+    function (err) {
+      apm.logger.error(err, 'useClientS3 rejected')
```

Comment:
Reviewer note: Before this change the error, if any, was silently not logged -- making debugging more difficult.